bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -74,16 +89,80 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     return py_arrays
 
 
+# Pythonizations for the following methods:
+# - ROOT.RDataFrame.Histo1D
+# - ROOT.RDataFrame.Histo2D
+# - ROOT.RDataFrame.Histo3D
+# - ROOT.RDataFrame.Profile1D
+# - ROOT.RDataFrame.Profile2D
+# Since a lot of their overloads take a ROOT.RDF.T*$DModel (where * can be
+# 'H' and 'Profile', while $ 1, 2 and 3) object as first argument,
+# we implement the possibility to pass the arguments of
+# ROOT.RDF.TH*DModel as a tuple in the first
+# argument of the five methods mentioned before.
+def pythonize_hp(self, partialmethod_list, *args):
+    # self: instantiation of RDataFrame
+    # partialmethod_list: list containing e.g. '_OriginalHisto1D' and
+    # e.g. the method Histo1D itself
+    # args: arguments passed by the user when he calls e.g Histo1D
+
+    # get the "original" method of the RDataFrame instantiation
+    original_function_hp = getattr(self, partialmethod_list[0])
+    # get the T*$DModel method
+    model_function = partialmethod_list[1]
+
+    if type(args[0]) == tuple:
+        # call the T*$DModel method with the elements of the tuple
+        # as arguments
+        model = model_function(*args[0])
+        # call the "model method" with model as first argument
+        if len(args) > 1:
+            res = original_function_hp(model, *args[1:])
+        else:
+            # covers the case of the overloads with only model passed
+            # as argument
+            res = original_function_hp(model)
+    # Fall back to the original implementation for the rest of the overloads
+    else:
+        res = original_function_hp(*args)
+
+    return res
+
+
 @pythonization()
 def pythonize_rdataframe(klass, name):
     # Parameters:
     # klass: class to be pythonized
     # name: string containing the name of the class
 
-    # Add AsNumpy feature
+    # Add AsNumpy feature and
+    # syntactic sugar for the following methods:
+    # - ROOT.RDataFrame.Histo1D
+    # - ROOT.RDataFrame.Histo2D
+    # - ROOT.RDataFrame.Histo3D
+    # - ROOT.RDataFrame.Profile1D
+    # - ROOT.RDataFrame.Profile2D
     if name.startswith("ROOT::RDataFrame<") or name.startswith("ROOT::RDF::RInterface<"):
         klass.AsNumpy = RDataFrameAsNumpy
 
+        # keys: names of the methods we want to pythonize
+        # values: T*$DModel method called by the function
+        methods_with_TModel = {'Histo1D' : RDF.TH1DModel, 'Histo2D' : RDF.TH2DModel, 'Histo3D' : RDF.TH3DModel, 'Profile1D' : RDF.TProfile1DModel, 'Profile2D' : RDF.TProfile2DModel}
```

Comment:
In order to avoid a long line and make it more readable, can you set the layout of the definition of this dictionary using a different line for every entry?